src/TCommandLine.cpp

```diff
@@ -382,7 +382,13 @@ bool TCommandLine::event(QEvent* event)
 #endif
                 // If EXACTLY Down is pressed without modifiers (special case
                 // for macOs - also sets KeyPad modifier)
-                historyMove(MOVE_DOWN);
+                bool bShouldClearInput = historyMove(MOVE_DOWN);
```

Comment:
Just a small style thing, the b prefix is not a pattern we use, would be nice if the naming was consistent with the rest and this was called `shouldClearInput`